content/legal/dekart-premium-subscription-agreement.md

```diff
@@ -0,0 +1,105 @@
+During the active subscription period, Licensee is granted a non-exclusive, non-transferable license to access and use the Dekart Premium codebase, subject to the terms of the **Dekart Premium Commercial License Agreement**.
+
+4.2 **Employee Access Limits**:
+Licensee may provide access to up to three (3) employees. Licensee is responsible for ensuring that these employees comply with this Agreement and any other applicable licenses.
+
+4.3 **Restrictions on Use**:
+Licensee may provide access to the **codebase**, **Docker images**, and **support services** within their organization. Licensee is responsible for ensuring that only authorized employees within the organization are granted access to the private Git repository and issue tracker (up to three employees, unless otherwise agreed). However, Licensee is free to copy and distribute the code and Docker images according to the terms of the **Dekart Premium Commercial License Agreement**.
+
+Licensee may not grant third parties (outside the Licensee's organization) access to the private Git repository, issue tracker, or support services unless explicitly authorized by Licensor. Any unauthorized sharing of access to the repository, issue tracker, or support services may result in immediate termination of access to the Service.
+
+## 5. **Termination of Subscription**
+
+5.1 **Termination by Licensee**:
+Licensee may terminate the subscription at any time through Stripe. Upon termination, access to the Service, including the Git repository and Docker images, will cease at the end of the current billing cycle.
+
+5.2 **Termination by Licensor**:
+Licensor reserves the right to terminate the subscription immediately if Licensee violates any terms of this Agreement or the **Dekart Premium Commercial License Agreement**.
+
+5.3 **Termination by Licensor for Convenience**:
+Licensor reserves the right to terminate this Agreement and the associated subscription at its sole discretion. In such cases, Licensor will provide notice to Licensee, and the Service will continue to be provided until the end of the current billing cycle. Licensee will not be charged for any subsequent billing periods following such termination.
+
+## 6. **Limitations of Liability**
+
+6.1 **No Liability for Data Loss**:
+Licensor is not responsible for any loss of data resulting from the use of the Service or the termination of the subscription. Licensee is responsible for backing up any code or data stored in the Git repository.
+
+6.2 **No Warranty**:
+The Service is provided "as is" without any warranty, express or implied, including but not limited to warranties of merchantability, fitness for a particular purpose, or non-infringement.
+
+## 7. **Governing Law and Dispute Resolution**
+
+This Agreement is governed by the laws of Germany, without regard to its conflict of law principles. Any disputes arising from this Agreement shall be resolved through arbitration in a neutral jurisdiction, as agreed by both parties.
```

Comment:
> in a neutral jurisdiction, as agreed by both parties - maybe it will be better to say that it will be handled in Berlin? At leas for our side it's better